mods/d2k/weapons/other.yaml

```diff
@@ -106,36 +106,51 @@ Demolish:
 		ImpactSounds: EXPLLG2.WAV
 		ImpactActors: false
 
-Atomic:
+DeathHand:
+	Warhead@Cluster: FireCluster
+		Weapon: DeathHandCluster
+		RandomClusterCount: 14
+		Dimensions: 3,3
+		Footprint: xxx xXx xxx
+	Warhead@2Eff: CreateEffect
+		Explosions: nuke
+		ImpactSounds: EXPLLG2.WAV
+		ImpactActors: false
+
+DeathHandCluster:
+	Inherits: Debris2
+	Range: 7c0
+	Projectile: Bullet
+		Image: 120mm
+		TrailImage: small_trail2
+		Speed: 96
+		LaunchAngle: 0, 32
+		Inaccuracy: 1c512
+		BounceCount: 0
 	Warhead@1Dam: SpreadDamage
-		Spread: 1c0
-		Falloff: 200, 100, 50, 25, 12, 0
-		Damage: 27000 ##225 in vanilla but of course is a cluster bomb instead, so damage spread out
+		Damage: 4500
```

Comment:
> tweaked both the DeathHand and ClusterMissile, including their damage (was way too low before, especially on the DH compared to Atreides airstrike).

Do you mean relative to the original game, or relative to common sense?

Reply:
Both. TS Cluster Missile was normally a guaranteed kill for at least up to Advanced Power Plants in the original, provided they were on the targeted cell.

Death Hand in the original had some inherent Missile inaccuracy, but when it hit something directly, it hit pretty hard.
Additionally, Ornithopters in our mod are more powerful than in the original, and actually feel a tiny bit more powerful than the DH even with this.
But to be honest, I'd prefer to leave any further D2k balance polishing to someone like @MustaphaTR, this was just to make the difference between the super weapons small enough that it doesn't make the DH a joke in comparison.